<launch>
    <!-- 定义参数，如果未设置则使用默认值 -->
    <arg name="gps_dbc_file" default="/home/nvidia/atv_ws/src/perception/config/CAN_.dbc" />
    <arg name="gps_can_channel" default="can0" />

    <!-- 启动节点，并传递参数 -->
    <node name="gps_imu_node" pkg="perception" type="gps_imu_node.py" output="screen">
        <param name="gps_dbc_file" value="$(arg gps_dbc_file)" />
        <param name="gps_can_channel" value="$(arg gps_can_channel)" />
    </node>

    <node name="wirecontrol" pkg="wirecontrol" type="wire_control_can_pub.py">
    </node>

</launch>
